Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './Footer.css';
+
+
+
+
+
+function FooterContainer(props) {
```

Comment:
This is a good opportunity for object destructuring; think about doing it like this:
```javascript
const FooterContainer = ({ RT, likes }) => {
  // ...
}; 
```